src/plugins/janus_videoroom.c

```diff
@@ -9459,6 +9470,7 @@ static void *janus_videoroom_handler(void *data) {
 						}
 						/* Are we subscribing to this publisher as a whole or only to a single stream? */
 						const char *mid = json_string_value(json_object_get(s, "mid"));
+						const char *opaqueid = json_string_value(json_object_get(s, "opaque_id"));
```

Comment:
Same here.

Reply:
Same here.
Am i missing something? It is added in the subscriber_stream_parameters object...